packages/gcc7.rb

```diff
@@ -0,0 +1,40 @@
+require 'package'
+
+class Gcc7 < Package
+  description 'The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Ada, and Go.'
+  homepage 'https://www.gnu.org/software/gcc/'
+  version '7.3.0'
+
+  source_url 'https://github.com/gcc-mirror/gcc/archive/gcc-7_3_0-release.zip'
+
+  source_sha256 'd8cb199f0d98bd83e3dac645126722e44137c24cbe5e87e1113dd4270f7530c6'
+
+  depends_on 'buildessential'
```

Comment:
Actually, you can remove this also.  It's a fake package that includes all the core dependencies anyway.

Reply:
Fixed